InteractionBase/InteractionBase.pro

```diff
@@ -78,7 +78,12 @@ HEADERS += src/commands/CommandWithFlags.h \
     src/commands/CToggleArrowLayer.h \
     src/commands/CAddInfoNode.h \
     src/handlers/HInfoNode.h \
-    src/commands/CToggleInfoEntry.h
+    src/commands/CToggleInfoEntry.h \
+    src/nodes/ViewSelectionNode.h \
```

Comment:
Please call this ViewSwitcherNode and the corresponding item VViewSwitcher. I'm actually curious to see why we need a node at all here.